BSP/Inc/BSP_Timer.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * Header file for the library to interact
+ * with the timers
+ */
+#include <stdio.h>
+
+#include <stdlib.h>
+
+/** 
+ * @brief   Updates the time by reading the CSV file and calling relevant functions at the right time 
+ * @param   none
+ * @return  none
+ */  
+void BSP_Timer_Update();
+
+/** 
+ * @brief   Intializes the Timer
+ * @param   2 reload values for two timers, Two function pointers 

```

Comment:
Good, but I would prefer it if you just split these up. The tools in most IDEs look for it split up. You can look at the simulator_can branch for an example of how that's done with a lot of parameters.

Reply:
Still this